cdap-ui/app/features/adapters/etlapps.less

```diff
@@ -67,7 +67,13 @@ body.theme-cdap.state-adapters-create {
         }
       }
       .template {
+        &.plugin-added {
+          background-color: #2c498b;
```

Comment:
Nit-picky, but is either of these background colors (lines 70, 75) a variable? If not, feel free to disregard.